src/applications/gi/sass/partials/_gi-vet-tec.scss

```diff
@@ -19,4 +19,12 @@
     margin: 0;
   }
 
+  @media (min-width: 641px) {
+    .single-column-display-block{
+      display: block !important; }
+    .single-column-display-none {
```

Comment:
Why are these custom classes being used when there are already [display classes](https://design.va.gov/utilities/display) defined?

Reply:
That is the width in which our UX has specified for the single column view on the search results page.